development/metamaskbot-build-announce.js

```diff
@@ -98,6 +102,21 @@ async function start() {
     },
   };
 
+  const commitMessage = (
+    await execFile('git', ['show', '-s', '--format=%s', HEAD_COMMIT_HASH])
```

Comment:
nit: If you _want_ to avoid the extra `promisify` and `await` steps `execSync` is probably fine here, we already use it in a bunch of other places in the build process.

That said, I wonder why we don't use tags for beta bulds, then we could check if the tag is a beta build instead of parsing the commit message.